task_12/style.css

```diff
@@ -0,0 +1,64 @@
+/* CSS Document */
+.grid {
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(400px, 1fr));
+    grid-gap: 10px;
+    grid-auto-flow: dense;
+}
+.menu-card {
+    box-shadow: 2px 2px 2px 2px #dbdbdb;
+    overflow: auto;
+    position: relative;
+}
+.span-2{
+    grid-column: span 2;
+}
+.menu-card__menu-image{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+.menu-card__menu-image__image{
```

Comment:
Incorrect element name name 
In this case will be better to use .menu-card__menu-image > img